src/components/backend-ai-environment-list.ts

```diff
@@ -1201,6 +1201,10 @@ export default class BackendAIEnvironmentList extends BackendAIPage {
       this._refreshSorter(e);
     });
 
+    document.addEventListener('image-list-refreshed', () => {
```

Comment:
The event name `image-list-refreshed` seems that the image list has been changed, but inside of the event listener it's calling `_getImages()`, which requests the images from manager server.
In my opinion, we could change the name of custom event to have closer meaning of the event , such as `registry-rescanned`.